apps/gitness/src/pages-v2/pull-request/pull-request-compare.tsx

```diff
@@ -243,19 +250,34 @@ export const CreatePullRequest = () => {
       setPrBranchCombinationExists(null)
     }
   }, [pullReqData])
+  const [query, setQuery] = useQueryState('query')
 
   // TODO:handle pagination in compare commit tab
-  const { data: { body: commitData } = {} } = useListCommitsQuery({
+  const { data: { body: commitData, headers } = {} } = useListCommitsQuery({
     repo_ref: repoRef,
 
     queryParams: {
+      // TODO: add query when commit list api has query abilities
+      // query: query??'',
```

Comment:
I dont think we support search on the commits page? 
https://harness.slack.com/archives/C07QA52CWKH/p1733869074973649?thread_ts=1733869048.973069&cid=C07QA52CWKH

Reply:
i just left it there in case we do support search on commit list api in the future